src/libraries/System.Data.Common/src/System/Data/ConstraintCollection.cs

```diff
@@ -624,7 +624,7 @@ internal int InternalIndexOf(string? constraintName)
             if ((null != constraintName) && (0 < constraintName.Length))
             {
                 int constraintCount = List.Count;
-                int result = 0;
+                int result;
```

Comment:
This can be declared in the loop where it's assigned.

Reply:
Done